Tests/SentryTests/Protocol/SentryEnvelopeTests.swift

```diff
@@ -162,6 +162,24 @@ class SentryEnvelopeTests: XCTestCase {
         XCTAssertEqual(eventId, envelopeHeader.eventId)
         XCTAssertEqual(traceContext, envelopeHeader.traceContext)
     }
+
+    func testInitSentryEnvelopeHeader_withSampleRand_SetIdAndTraceState() {
```

Comment:
`m`: What extra value does this test add? There aren't any changes in `SentryEnvelopeHeader`. We already have `testInitSentryEnvelopeHeader_SetIdAndTraceState` which validates that the init method of `SentryEnvelopeHeader` assigns the traceContext. Here, we only use a different initializer of the `TraceContext`. Am I missing something?

Reply:
You are correct, thanks for pointing this out. I'll remove the added test